src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
It is good practice to make the modifier of this variable private. A getter and a setter for explicitly changing the value of this variable are already in the class. This enhances encapsulation.